README.md

```diff
@@ -6,27 +6,106 @@
     <img src="https://codecov.io/gh/awslabs/amazon-s3-find-and-forget/branch/master/graph/badge.svg">
 </h1>
 
-> Warning: This project is currently in beta and should not be used for
-> production purposes.
-
 Amazon S3 Find and Forget is a solution to the need to selectively erase records
 from data lakes stored on Amazon Simple Storage Service (Amazon S3). This
 solution can assist data lake operators to handle data erasure requests, for
 example, pursuant to the European General Data Protection Regulation (GDPR).
 
-The Amazon S3 Find and Forget solution can be used with Parquet-format data
-stored in Amazon S3 buckets. Your data lake is connected to the solution via an
-AWS Glue table and specifying which columns in the table contain user
-identifiers.
+The solution can be used with Parquet-format data stored in Amazon S3 buckets.
+Your data lake is connected to the solution via AWS Glue tables and by
+specifying which columns in the tables contain user identifiers.
 
 Once configured, you can queue record identifiers that you want the
 corresponding data erased for. You can then run a deletion job to remove the
 data corresponding to the records specified from the objects in the data lake. A
 report log is provided of all the S3 objects modified.
 
+## Installation
+
+The solution is available as an AWS CloudFormation template and should take
+about 20 to 40 minutes to deploy. See the
+[deployment guide](docs/USER_GUIDE.md#deploying-the-solution) to launch the
+deployment with just one click and the
+[cost overview guide](docs/COST_OVERVIEW.md) to learn about its costs.
+
+> Warning: This project is currently in beta release
+>
+> - Actively developed by AWS
+> - Carefully review code and documentation if you intend to use the solution
+>   for production purposes
+> - Get in touch with the team via
+>   [Github Issues](https://github.com/awslabs/amazon-s3-find-and-forget/issues)
+>   for providing feedback and to ask any question. We appreciate your help
+
+## Usage
+
 The solution provides a web user interface, and a REST API to allow you to
 integrate it in your own applications.
 
+![HomePage Screenshot](docs/images/home-screenshot.png)
```

Comment:
Not sure about this one. The blogpost may have more details about this, so for now I just put one screenshot. But I can also remove it if we think it doesn't add any value

Reply:
That screenshot on its own doesn't feel very valuable to me.

I have no problem with adding it now, but I think we should improve it ... maybe experiment with creating an animated GIF that shows the typical workflow?

Reply:
I think you're right. I removed it for now, let's try making a gif in a separate PR